### ISSUE TITLE
Bind a resource programatically

### ISSUE THREAD

**Issue by muliyul** [dropwizard/dropwizard#3650]
Trying to bind a programatically created resource results in this error

>WARN  [2021-01-13 03:13:22,564] org.glassfish.jersey.internal.inject.Providers: A provider org.glassfish.jersey.server.model.Resource registered in SERVER runtime does not implement any provider interfaces applicable in the SERVER runtime. Due to constraint configuration problems the provider org.glassfish.jersey.server.model.Resource will be ignored. 

```kotlin
val resource = Resource.builder(uriPath.removeSuffix("/")).apply {
	addMethod(HttpMethod.GET)
		.consumes(MediaType.WILDCARD)
		.produces(MediaType.TEXT_HTML)
		.handledBy { return@handledBy configuredIndex }
}
.build()
environment.jersey().register(resource)
```

Is it possible to register a resource programatically?

**Comment by zUniQueX** [comment]
According to the [Jersey documentation](https://eclipse-ee4j.github.io/jersey.github.io/documentation/latest/resource-builder.html), this can be done with the `registerResources(...)` method.

A minimal example in Dropwizard would then look like this:
```java
Resource.Builder builder = new Resource.builder("/");
builder.addMethod(HttpMethod.GET).handledBy(cRC -> "Hello World!");
environment.jersey().getResourceConfig().registerResources(builder.build());
```

**Comment by zUniQueX** [comment]
@muliyul did this solve it for you? Then I think this issue isn't needed any more.

**Comment by muliyul** [comment]
I moved on to other things. Haven't tested it though.

**Comment by zUniQueX** [comment]
@muliyul I don't think this use case is common enough to make it to our docs. If you have no issue with this any more, I'd request you to close this.

**Comment by muliyul** [comment]
I did not request to add it to the docs, rather ask if supported.
